test/compilable/depsOutput9948.d

```diff
@@ -1,12 +1,22 @@
-// PERMUTE_ARGS:
-// REQUIRED_ARGS: -deps=${RESULTS_DIR}/compilable/depsOutput9948.deps
-// POST_SCRIPT: compilable/extra-files/depsOutput.sh 
-// EXTRA_SOURCES: extra-files/depsOutput9948a.d
+/*
+PERMUTE_ARGS:
+REQUIRED_ARGS: -deps
+TRANSFORM_OUTPUT: remove_lines("druntime")
+TEST_OUTPUT:
+---
+depsImport depsOutput9948 (compilable$?:windows=\\|/$depsOutput9948.d) : private : imports.depsOutput9948a (compilable$?:windows=\\|/$imports$?:windows=\\|/$depsOutput9948a.d)
```

Comment:
Not entirely happy about all these condition sequences to handle `/` vs `\\`.
Maybe it's time for a `replace` transformation?